examples/inference/cit_example/run_jester.sh

```
@@ -0,0 +1,26 @@
+# Wrapper script — alternative to the absolute-path approach in submit.sub.
+# Use this as the executable if you need full venv environment activation
+# (e.g. if run_jester_inference reads VIRTUAL_ENV or other venv variables).
+#
+# To use: set in submit.sub:
+#   executable          = run_jester.sh
+#   transfer_executable = true
+
+set -e
+
+VENV=/home/thibeau.wouters/jester_analyses/jester/.venv
+source "${VENV}/bin/activate"
```

Shellcheck (0.11.0)

[info] Not following: ./bin/activate was not specified as input (see shellcheck -x).

(SC1091)